bolt-expo/app.json

```diff
@@ -13,7 +13,7 @@
     },
     "web": {
       "bundler": "metro",
-      "output": "static",
+      "output": "single",
```

Comment:
Does the deployment to Netlify still work with this?

Reply:
I think it does. With this the behaviour is:

> `single` outputs a Single Page Application (SPA), with a single index.html in the output folder, and has no statically indexable HTML

(docs: https://docs.expo.dev/versions/latest/config/app/#output)